lib/iris/experimental/um.py

```diff
@@ -687,6 +687,18 @@ def normalise(values, kind):
             sector_size = self._WORDS_PER_SECTOR * self._word_size
             for field in self.fields:
                 data = field.get_data()
+
+                if hasattr(field, "lbpack"):
+                    lbpack = field.lbpack
+                    packing = lbpack % 10
+                    compression = (lbpack // 10) % 10
+                    number_format = (lbpack // 1000) % 10
+                    if (packing not in (0, 1) or
```

Comment:
I don't currently have a way of checking whether data is being passed-through, so I'm guessing that this can't be merged at the moment as it would break what @pp-mo is doing. Looking at tests this morning, hopefully.